content/python/concepts/strings/strings.md

```diff
@@ -111,7 +120,7 @@ print(note)
 
 ## Modifying Strings
 
-Python has special operators to modify strings. For example, we can use `+` to concatenate two strings or `*` to duplicate a string. We can also use the keyword `in` to see if a given character exists in a `string`.
+Python has special operators to modify strings. For example, we can use `+` to concatenate two strings, and `*` to duplicate a string. We can also use the keyword `in` to see if a given character exists in a `string`.
```

Comment:
For this line and line 143 (I know using "we" was not your choice), it should be declarative:

"For example, `+` can be used to concatenate two strings and..."

Reply:
OK, will fix this later along with the change to line 104.